packages/ui/src/ui/pages/query-tracker/module/query_aco/actions.ts

```diff
@@ -0,0 +1,43 @@
+import {ThunkAction} from 'redux-thunk';
+import {Action} from 'redux';
+import {YTApiId, ytApiV4Id} from '../../../../rum/rum-wrap-api';
+import type {ActionD} from '../../../../types';
+import type {QueryACOState} from './reducer';
+import {getQueryTrackerRequestOptions} from '../query/selectors';
+
+export const QUERY_ACO_LOADING = 'query-tracker/QUERY_ACO_LOADING';
+export type QueryACOLoadingAction = Action<typeof QUERY_ACO_LOADING>;
+
+export const QUERY_ACO_ERROR = 'query-tracker/QUERY_ACO_ERROR';
+export type QueryACOLoadingError = Action<typeof QUERY_ACO_ERROR>;
+
+export const QUERY_ACO_SUCCESS = 'query-tracker/QUERY_ACO_SUCCESS';
+export type QueryACOLoadingSuccess = ActionD<
+    typeof QUERY_ACO_SUCCESS,
+    QueryACOState['access_control_objects']
+>;
```

Comment:
it is better to put action-constants to a separate file. And we have `createActionTypes` helper function to create such constants.